### PR TITLE
Fix chromecast

### PR DESCRIPTION
**Pull request by Marius456**
## Description

- Fix chromecast so it could run in template starter

## Related issues

- https://github.com/flexn-io/renative/issues/1456

## Npm releases

n/a

### CODE REVIEW

**Comment by GabrieleKaceviciute** [comment]
@RicardasN 
no available devices found ->
chromecast_and

although e.g. youtube - finds it ->
youtube_cast

**Comment by RicardasN** [comment]
@GabrieleKaceviciute @pauliusguzas Could you guys verify that it was now fixed on Android? Haven't touched anything in regards to iOS, so expect no changes there.

**Comment by GabrieleKaceviciute** [comment]
Note: no crashes on ios for me

**Comment by RicardasN** [comment]
> @RicardasN all platforms that do not support cast function have info about it -> macos
> 
> but androidtv and firetv does not -> aTV
> 
> is it expected?

No, it should also return text, will check it out

**Comment by RicardasN** [comment]
@GabrieleKaceviciute fixed

**Review by GabrieleKaceviciute — APPROVED** [review]

File: packages/app-harness/src/app/index.tsx [hunk 1]
```diff
@@ -132,6 +133,9 @@ const AppContent = () => {
                     <TestCase id={5} title="Image Support">
                         <Image source={ICON_LOGO} style={{ width: 100, height: 100 }} />
                     </TestCase>
+                    <TestCase id={6} title="Cast Support">
```
Comment (aurimasmi) [inline packages/app-harness/src/app/index.tsx:136]: Test id 6 is same as below

File: packages/app-harness/src/components/CastButton/index.android.mobile.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { CastButton, useRemoteMediaClient } from 'react-native-google-cast';
+
+export function CastComponent() {
+    const client = useRemoteMediaClient();
+
+    if (client) {
```
Comment (aurimasmi) [inline packages/app-harness/src/components/CastButton/index.android.mobile.tsx:7]: is this component re-renders for some side reason is having this in render cycle wont break anything? better to put it in useEffect?

File: packages/app-harness/src/config.tsx [hunk 3]
```diff
@@ -27,3 +29,98 @@ if (typeof global.performance.now !== 'function') {
 }
 
 export { CONFIG, ICON_LOGO };
+
+const getFlexPropertyValue = () => {
```
Comment (aurimasmi) [inline packages/app-harness/src/config.tsx:33]: is function really neccesary for this single value?

File: packages/app-harness/renative.json [hunk 4]
```diff
@@ -165,6 +165,17 @@
             "android": {
                 "implementation": "// TEST"
             }
+        },
+        "react-native-google-cast": {
```
Comment (aurimasmi) [inline packages/app-harness/renative.json:169]: Expected to be disable for those platforms?

File: packages/app-harness/src/components/CastButton/index.android.mobile.tsx (hunk 2 above)
Comment (RicardasN) [reply packages/app-harness/src/components/CastButton/index.android.mobile.tsx:7]: https://react-native-google-cast.github.io/docs/getting-started/usage
Fair enough, will update that, did it according to documentation example of usage

File: packages/app-harness/renative.json (hunk 4 above)
Comment (RicardasN) [reply packages/app-harness/renative.json:169]: yes

**Pull request merged**